hawk_eye_notify/__main__.py

```diff
@@ -0,0 +1,47 @@
+docstr = """hawk_eye_notify
+Hawk_Eye_Notify will run a single script with as many xargs as are passed to it.
+If the job being run requires multiple scripts, design one script to accept the args
+needed to run the others and run that one.
+
+One thing to note is that the script will be ran from root,
```

Comment:
This means the root path but I initially thought that the script would be run as the root user